### PR TITLE
Add ep 173

### PR DESCRIPTION
**Pull request by adnanmerrakchi**
add ep 173

### CODE REVIEW

File: blablas/ep173/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: "#173 - Blanat Coding Challenge #BCC discussion"
+tags: ["DEV"]
+category: "DEV"
+isNext: false
+youtube: https://www.youtube.com/watch?v=gOnapDngTuk
+published: true
+featured: false
+---
+
+In this episode, we discuss how to prepare for talks, taking 3 months long vacations and the future of the tech industry in 2024.
```
Comment (yjose) [inline blablas/ep173/index.md:14]: @adnanmerrakchi  apparently you missed updating the description

File: blablas/ep173/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+date: 2024-03-10
+time: 21h:00min
+duration: "1:29:29"
+title: "#173 - Blanat Coding Challenge #BCC discussion"
+tags: ["DEV"]
+category: "DEV"
```
Comment (yjose) [inline blablas/ep173/index.md:7]: dev should category should be lowercase

File: blablas/ep173/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+date: 2024-03-10
+time: 21h:00min
+duration: "1:29:29"
+title: "#173 - Blanat Coding Challenge #BCC discussion"
```
Comment (yjose) [inline blablas/ep173/index.md:5]: no need for episode number

File: blablas/ep173/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+- [challenge shared by Saad](https://codecup.nl/http://espoir-comp...)
+
+- [The runner that Mehdi is describing](https://github.com/geeksblabla/blarun)
+
+- [Challenge link](https://github.com/geeksblabla/blanat)
+
+## Prepared and Presented by
+
+- [Adnan MERRAKCHI]
```
Comment (yjose) [inline blablas/ep173/index.md:54]: should be a link, twitter or LinkedIn

**Review by yjose — CHANGES_REQUESTED** [review]

File: blablas/ep173/index.md (hunk 1 above)
Comment (adnanmerrakchi) [reply blablas/ep173/index.md]: done

File: blablas/ep173/index.md (hunk 2 above)
Comment (adnanmerrakchi) [reply blablas/ep173/index.md]: done

File: blablas/ep173/index.md (hunk 3 above)
Comment (adnanmerrakchi) [reply blablas/ep173/index.md]: done

File: blablas/ep173/index.md (hunk 4 above)
Comment (adnanmerrakchi) [reply blablas/ep173/index.md]: done

**Review by yjose — APPROVED** [review]

**Pull request merged**